eng/Versions.props

```diff
@@ -77,7 +77,7 @@
     <MicrosoftWin32SystemEventsPackageVersion>4.6.0</MicrosoftWin32SystemEventsPackageVersion>
     <SystemComponentModelAnnotationsPackageVersion>4.6.0</SystemComponentModelAnnotationsPackageVersion>
     <SystemDiagnosticsEventLogPackageVersion>4.6.0</SystemDiagnosticsEventLogPackageVersion>
-    <SystemDrawingCommonPackageVersion>4.6.2</SystemDrawingCommonPackageVersion>
+    <SystemDrawingCommonPackageVersion>4.6.0</SystemDrawingCommonPackageVersion>
```

Comment:
Why did this decrease?

Reply:
Coherency bug just recently fixed in Darc but not rolled out to Maestro yet.  I'll fix it up